tasks/letsencrypt.yml

```diff
@@ -0,0 +1,36 @@
+# tasks file for haproxy
+---
+- name: letsencrypt | copy SSL deploy script
+  template:
+    src: "{{ haproxy_letsencrypt_ssl_deploy_template }}"
+    dest: "{{ haproxy_letsencrypt_ssl_deploy }}"
+    owner: root
+    group: root
+    mode: 0755
+  tags:
+    - haproxy-letsencrypt-ssl-deploy
+
+- name: letsencrypt | copy OCSP deploy script
+  template:
+    src: "{{ haproxy_letsencrypt_ocsp_deploy_template }}"
+    dest: "{{ haproxy_letsencrypt_ocsp_deploy }}"
+    owner: root
+    group: root
+    mode: 0755
+  tags:
+    - haproxy-letsencrypt-ocsp-deploy
+
+- name: letsencrypt | configure (cron) job for OCSP deploy
+  cron:
+    name: haproxy-letsencrypt-ocsp-deploy
+    job: "{{ haproxy_letsencrypt_ocsp_deploy }}"
+    state: "{{ haproxy_letsencrypt_ocsp_deploy_job.state | default('absent') }}"
```

Comment:
Most of the times, no `state` means `present`. Is this on purpose?

Reply:
`present` is better, but that will create the job when there is no (custom) configuration. Do you want that?

Reply:
Probably, you're right